profiles/src/main/java/software/amazon/awssdk/profiles/ProfileSystemSetting.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.profiles;
+
+import java.nio.file.Paths;
+import software.amazon.awssdk.annotations.ReviewBeforeRelease;
+import software.amazon.awssdk.profiles.internal.ProfileFileLocations;
+import software.amazon.awssdk.utils.JavaSystemSetting;
+import software.amazon.awssdk.utils.SystemSetting;
+
+/**
+ * The system properties and environment variables supported by the AWS SDK.
+ *
+ * @see JavaSystemSetting
+ */
+@ReviewBeforeRelease("Do we need all of these, or can they be controlled in the client configuration?")
```

Comment:
Do you think we should still have this review-before-release?

Reply:
nope, removed.